components/ipfs/ipfs_utils_unittest.cc

```diff
@@ -657,3 +684,71 @@ TEST_F(IpfsUtilsUnitTest, IsAPIGatewayTest) {
   ASSERT_FALSE(ipfs::IsAPIGateway(GURL("https://brave.com"), channel));
   ASSERT_FALSE(ipfs::IsAPIGateway(GURL(), channel));
 }
+
+TEST_F(IpfsUtilsUnitTest, ParseIPFSUri) {
+  GURL url("ipfs://bafy/path");
```

Comment:
I would pack this checks into an utility function, and basically check all getters, just to set our expectations about such GURLs. `GetOrigin()`, `EffectiveIntPort()` because they check whether the url is Standard

Reply:
done